nvtabular/dispatch.py

```diff
@@ -167,3 +167,20 @@ def _to_arrow(x):
         return x.to_arrow()
     else:
         return pa.Table.from_pandas(x, preserve_index=False)
+
+
+def _concat(objs, **kwargs):
+    if isinstance(objs[0], (cudf.DataFrame, cudf.Series)):
+        return cudf.core.reshape.concat(objs, **kwargs)
+    else:
+        return pd.concat(objs, **kwargs)
+
+
+def _make_df(x=None, device=None):
```

Comment:
elsewhere in this file we're using ```_like_df``` to indicate what type of dataframe it should be based off (see like _arange etc). Can we go

```suggestion
def _make_df(_like_df=None, device=None):
```

to be consistent?